src/content.js

```diff
@@ -24,6 +24,7 @@ import { t } from "./i18n";
  * @property {string} text
  * @property {string} fontFamily
  * @property {number} fontWeight
+ * @property {string} fontStyle
```

Comment:
Nitpick, since this is optional:
```suggestion
 * @property {string} [fontStyle]
```

Out of curiosity? Did you edit `types/content.d.ts` manually, or did you run `npm run typescript`?

Reply:
manually :0